src/List/ListGroup.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ListGroup } from './ListGroup';
+
+export default {
+  title: 'Components/List',
+  component: ListGroup,
+  argTypes: { onClick: { action: 'clicked' } },
+  parameters: {
+    backgrounds: {
+      default: 'orange',
+    },
+  },
+} as Meta;
+
+const Template: Story = ({ children, ...args }) => (
+  <ListGroup {...args}>
+    {children.map((child: React.ReactNode) => (
+      <ListGroup.Item key={Math.random()}>{child}</ListGroup.Item>
+    ))}
+  </ListGroup>
+);
+
+export const Controlled = Template.bind({});
+Controlled.args = {
+  disabled: false,
+  children: ['First', 'Second', 'Third'],
+};
+
+export const ControlledListItem = Template.bind({});
+ControlledListItem.args = {
+  disabled: false,
+  children: ['This is a list item'],
+};
+
+export const DisabledListItem = Template.bind({});
+DisabledListItem.args = {
+  disabled: true,
+  children: ['I am disabled'],
+};
+
+export const DefaultList: Story = args => (
+  <ListGroup {...args}>
+    <ListGroup.Item key={Math.random()}>
+      <ControlledListItem {...ControlledListItem.args} />
+      <DisabledListItem {...DisabledListItem.args} />
+    </ListGroup.Item>
+  </ListGroup>
+);
+
+export const SingleItemList: Story = args => (
+  <ListGroup {...args}>
+    <ListGroup.Item key={Math.random()}>
+      <ControlledListItem {...ControlledListItem.args} />
+    </ListGroup.Item>
```

Comment:
These will be deleted but worth watching out for this, this nest a list item in a list item and results in the extra padding you have in these 2 stories

Reply:
Ah, yes, I had a separate template for these and missed this when I removed it.